encodings/runend/src/array.rs

```diff
@@ -237,6 +239,11 @@ impl StatisticsVTable<RunEndArray> for RunEndEncoding {
                     .unwrap_or(false)
                     && array.logical_validity().all_valid(),
             )),
+            Stat::TrueCount => match array.dtype() {
```

Comment:
I would move all the statistics logic to separate file

Reply:
done.